ansible/roles/stack-sunbird/templates/sl-assessments.env

```diff
@@ -0,0 +1,97 @@
+# Service Config
+# Port on which the service will listen to
+PORT={{sl_assessments_port | default("7201")}} 
+
+IMPROVEMENT_PROJECT_BASE_URL
+#Application ENV example local/dev/production
+NODE_ENV={{sl_assessments_node_env}}
+
+#Log configuration
+#Enable console logging
+ENABLE_DEBUG_LOGGING={{sl_assessments_enable_debug_logging | default("ON")}}
+
+#Enable writing logs to a file
+ENABLE_BUNYAN_LOGGING={{sl_assessments_enable_bunyan_logging | default("OFF")}}
+
+#service based URL
+APPLICATION_BASE_URL={{sl_assessments_application_base_url | default('/assessment/')}}
+
+# Allow report request to run for longer time
+request_timeout_for_reports={{sl_assessments_request_timeout_for_reports | default("600000")}}
+SHIKSHALOKAM_USER_PROFILE_FETCH_ENDPOINT={{sl_assessments_shikshalokam_user_profile_fetch_endpoint | default("/api/user/v1/read")}}
+AUTHORIZATION={{sl_assessments_authorization}}
+MONGODB_URL={{sl_assessments_mongodb_url}}
+SHIKSHALOKAM_BASE_HOST={{sl_assessments_shikshalokam_base_host}}
+
+## Cloud config
+# Cloud Storage Configuration
+CLOUD_STORAGE={{sl_assessments_cloud_storage}}
+
+# Azure Cloud Configuration
+AZURE_ACCOUNT_NAME={{sl_assessments_azure_account_name}}
+AZURE_ACCOUNT_KEY={{sl_assessments_azure_account_key}}
+AZURE_STORAGE_CONTAINER={{sl_assessments_azure_storage_container)}}
+
+# Current in use database name   
+DB={{sl_assessments_db}}
+
+# Secure token access Admin related API's
+INTERNAL_ACCESS_TOKEN={{sl_assessments_internal_access_token}}
+
+# Disable learner service check
+DISABLE_LEARNER_SERVICE_ON_OFF={{sl_assessments_disable_learner_service_on_off}}
```

Comment:
why do we have to check learner service health or disable it? What should be the default behaviour ?

Reply:
default behaviour is on i will add it

Reply:
changes done